### ISSUE TITLE
Chord Recognition Train/Eval Code

### ISSUE THREAD

**Issue by burstMembrane** [MTG/omar-rq#63]
I'm looking into using OMAR-RQ for chord recognition. 
I noticed the README + Paper reports chord accuracy results  but `chords.gin` appears to be a placeholder. Could you share the evaluation/training code/config you used for the chord prediction, or point me to the correct branch? Thanks, really interesting project!

**Comment by andreamust** [comment]
Thanks for your interest in the project! I'm working on refining and pushing the code for the chord recognition downstream task, which should be up soon. 

In the meantime, for the evaluation pipeline you can refer to https://github.com/andreamust/consonance-ACE; the chord estimation evaluation code there is very similar to what we used for the experiments documented in the paper.

We will be notifying you in this issue as soon as the code is uploaded to the repo.

**Comment by burstMembrane** [comment]
Thank you @andreamust, I will keep an eye on it here and check out the eval repo in the meantime. Thanks for this project, very interesting work!

**Comment by zhishengz-nv** [comment]
Hi @andreamust, thanks for open-sourcing this amazing work. Is there any estimated date for releasing checkpoints for chord recognition?
